tests/general/KeywordBearTest.py

```diff
@@ -185,3 +185,32 @@ def test_keyword_between_code(self):
                              '+/*\n'
                              ' test\n'
                              ' */\n')
+
+    def test_keyword_regex(self):
+        text = ['# add two given values and result the result\n',
+                'def add(a, b):',
+                '    return a+b\n',
+                '               \n',
+                'print(add(2, 3))\n']
+
+        regex_keyword = 'r.s.l.'
+
+        with execute_bear(self.uut, filename='F', file=text,
+                          regex_keyword=regex_keyword,
+                          dependency_results=self.dep_results) as result:
+            self.assertEqual(result[0].message, 'The line contains the keyword'
+                                                " 'result' which resulted in a"
```

Comment:
No. We talked about the quoting style in the maintainers channel and apparently this is the preferred way.